Libraries.md

```diff
@@ -2,151 +2,151 @@
+	A fast circular ImageView perfect for profile images. As this is just a custom ImageView and not a custom 	Drawable, or a combination of both, it can be used with all kinds of drawables, i.e., a PicassoDrawable 	from Picasso or other non-standard drawables.
 
-	More info with code examples is available [here.](https://github.com/hdodenhof/CircleImageView)
+	More info with code examples is available [here](https://github.com/hdodenhof/CircleImageView).
 
 4. **Android Crop**
 
-	AndroidCrop provides a simple image cropping Activity, based on code from AOSP. It's backwards compatible to Gingerbread and has nice and clean UI.
+	Android Crop provides a simple image cropping Activity, based on the code from AOSP. It's backwards compatible with Gingerbread and has nice and clean UI.
 
-	More info with code examples is available [here.](https://github.com/jdamcd/android-crop)
+	More info with code examples is available [here](https://github.com/jdamcd/android-crop).
 
 5. **Holo Graph Library**
 
 	Slick, easy-to-use graphs with click listeners.
 
-	More info with code examples is available [here.](https://github.com/Androguide/HoloGraphLibrary)
+	More info with code examples is available [here](https://github.com/Androguide/HoloGraphLibrary).
 
 6. **Glide**
 
-	Glide is a fast and efficient open source media management and image loading framework for Android that wraps media decoding, memory and disk caching, and resource pooling into a simple and easy to use interface. It also has support for animated GIF's.
+	Glide is a fast and efficient open source media management and image loading framework for Android, that wraps media decoding, memory and disk caching, and resource pooling into a simple and easy-to-use interface. It also has support for animated GIFs.
 
-	More info with code examples is available [here.](https://github.com/bumptech/glide)
+	More info with code examples is available [here](https://github.com/bumptech/glide).
 
 7. **Android Support Library package**
 
-	Support Library package is a set of official Google libraries used for backwards compatibility with older API. Each of these libraries supports a specific range of Android platform versions and set of features.
+	The Support Library package is a set of official Google libraries used for backwards compatibility with older APIs. Each of these libraries supports a specific range of Android platform versions and set of features.
 
-	More info and specific documentation about support library package is available [here.] (https://developer.android.com/tools/support-library/features.html)
+	More info and specific documentation about the Support Library package is available [here] (https://developer.android.com/tools/support-library/features.html).
```

Comment:
What should be the best way to define links. I see above it was changed so the dot is part of the link name *here.* and now and below it is kicked outside. I personally prefer outside but don't have any strong feeling about it. Just to make it consistent.

Reply:
I agree that the period should be outside, as it's part of the sentence and not the link (so it shouldn't be highlighted in blue and underlined).